eox_tagging/api/v1/viewset.py

```diff
@@ -36,6 +37,32 @@ def get_queryset(self):
 
         return queryset
 
+    def create(self, request, *args, **kwargs):
+        """"Hijack the create method and use a wrapper function to perform the
+        audit process. The original parameters of create are not very useful in
+        raw form, this way we pass more useful information to our wrapper
+        function to be audited
+        """
+
+        @audit_method(action="eox_tagging:viewset-create")
```

Comment:
could you be more specific, for example api-v1-tag-viewset, if we add a new api version or a new viewset we will be able to know the specific viewset

Reply:
Done